distributed/tests/test_active_memory_manager.py

```diff
@@ -431,21 +440,25 @@ async def test_drop_with_paused_workers_with_running_tasks_1(c, s, a, b):
     b is running and has no dependent tasks
     """
     x = (await c.scatter({"x": 1}, broadcast=True))["x"]
-    y = c.submit(slowinc, x, delay=2, key="y", workers=[a.address])
+    y = c.submit(slowinc, x, delay=2.5, key="y", workers=[a.address])
+
     while "y" not in a.tasks or a.tasks["y"].state != "executing":
         await asyncio.sleep(0.01)
     a.memory_pause_fraction = 1e-15
     while s.workers[a.address].status != Status.paused:
         await asyncio.sleep(0.01)
-    assert s.tasks["y"].state == "processing"
```

Comment:
Why remove this? I'd expect this to be true iff the next line is true as well?

Reply:
processing->memory transitions always happen on the worker first. It seemed redundant to test it on both scheduler and worker since we're not testing the state machine here.